var/spack/repos/builtin/packages/ecp-data-vis-sdk/package.py

```diff
@@ -155,7 +155,7 @@ class EcpDataVisSdk(BundlePackage, CudaPackage, ROCmPackage):
                        when='+visit',
                        propagate=['hdf5', 'adios2'])
 
-    dav_sdk_depends_on('vtk-m@1.7:+shared+mpi+openmp+rendering',
```

Comment:
Leave this I think, we want to be able to build a vtk-m 1.8 with ROCm if ascent is turned off.